src/status_im/ui/screens/events.cljs

```diff
@@ -370,7 +374,7 @@
                          [:process-pending-messages]
                          [:update-wallet]
                          [:update-transactions]
-                         [:get-fcm-token]
+                         (if-not platform/desktop? [:get-fcm-token])
```

Comment:
I'd probably rather have an explicit method for this `platform/mobile?` rather then checking that is not desktop (the implementation might very well be `not desktop?`)